src/views/collection-view/index.tsx

```diff
@@ -28,12 +28,11 @@ function CollectionView({
       }
     >
       <CollectionMeta
-        heading={{ text: name, id: layoutProps.headings[0].slug }}
+        heading={{ text: name, id: 'overview' }}
```

Comment:
Hard-coded this at this level, as `slug` in the underlying `Heading` component is a required prop.

Reply:
You could also use `id` or `slug` from the `collection` prop

Reply:
ah true, that might make it more clear that this now has nothing to do with sidecar - will update! 👍